anvil/src/udev.rs

```diff
@@ -716,9 +751,55 @@ fn scan_connectors(
             let allocator =
                 GbmAllocator::new(gbm.clone(), GbmBufferFlags::RENDERING | GbmBufferFlags::SCANOUT);
 
+            #[cfg(feature = "egl")]
+            let (render_feedback, scan_out_feedback) = {
+                // TODO: We should use the primary node as the main device and
+                // add the renderer as a preferred target, but as the gpu manager is
+                // late bound we can't do that here
+                let render_feedback =
+                    DmabufFeedbackBuilder::new(render_node.dev_id(), render_formats.clone())
```

Comment:
I think those should be `texture_formats`? (We are probably doing it wrong in the `cfg(not(feature = "egl"))` case as well.)

For rendering we need to sample from the provided dmabufs, no render into them.
It would really be nice, if we could use `ImportDma::supported_formats` instead, as that is more readable/obvious, but I get why that isn't possible.

Reply:
Good catch, I added the texture formats but forgot to use them. I think the non egl case should be fine as we only need the render formats for initializing the surface.

Reply:
refactored because of late init which now uses the renderer dmabuf formats